src/bootstrap/src/lib.rs

```diff
@@ -98,6 +98,8 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &str, Option<&[&'static str]>)] = &[
     /* Extra values not defined in the built-in targets yet, but used in std */
     (Some(Mode::Std), "target_env", Some(&["libnx"])),
     // (Some(Mode::Std), "target_os", Some(&[])),
+    (Some(Mode::Std), "target_os", Some(&["zkvm"])),
```

Comment:
Could you add those cfg bootstrap so that the release can remove them in the next bootstrap beta bump.
```suggestion
    // #[cfg(bootstrap)] zkvm
    (Some(Mode::Std), "target_os", Some(&["zkvm"])),
```